src/app/angular-carbonldp/guards.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./guards/authenticated.guard";
```

Comment:
Why use the name `angular-carbonldp` for the directory? It seems as if the things inside the old package were just copied to this project. Instead, put each file where they should belong to.